src/camera.h

```diff
@@ -93,7 +93,7 @@ extern Matrix gOrthoMatrix;
 
 extern u8 D_800DD2F8[8];
 
-extern s32 D_800DD300;
+extern s32 sNoControllerPluggedIn;
```

Comment:
Thoughts on labelling this as a global if this extern is necessary, even if it is technically only used in camera.c?

Reply:
Not needed to make it extern, probably got left behind during some other work before. I've removed it now. The only benefit to putting externs in the header is it helps when generating the ctx.c context for mips 2 c, but then the argument would have to be made for ALL variables to be put in there, which seems excessive. Gone.